sensor_msgs/Image[] video_frames
---
std_msgs/String activity